packages/api-server/src/middleware/withWebServer.ts

```diff
@@ -19,8 +29,10 @@ const withWebServer = (app: Application) => {
   )
 
   // For SPA routing on unmatched routes
-  app.get('*', function (_, response) {
-    response.send(indexContent)
+  Object.keys(contents).forEach((content) => {
+    app.get(`/${content}`, function (_, response) {
+      response.send(contents[content])
```

Comment:
Do you think its worth setting the content type header here?

Reply:
I'm conflicted about what the value should be for prerendering a javascript application with html.  Do you have opinions?

Reply:
I think its just plain html :) `Content-Type: text/html; charset=UTF-8`